src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs

```diff
@@ -66,6 +66,16 @@ public void ValidFileOptions(FileOptions option)
             }
         }
 
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
Ah, ok, makes sense. @JeremyKuhne, do you know why the existing test is excluded on Nano? We at least need a comment next to that saying why.